Hc.Gosia/Hc.Gosia.Js.Ex3/actions.js

```diff
@@ -0,0 +1,50 @@
+﻿var actions = actions || {};
+
+(function () {
+    function hideElement(id) {
+        document.getElementById(id).hidden = 'hidden';
+    }
+    actions.hideElement = hideElement;
```

Comment:
wszytskio pięknie, ale mam fajny tip :) jak tworzysz funkcje i chcesz ją udostępnić to można zrobić od razu przypisanie do globalnego obiektu np. actions.hideElement = function() {... }